static/app/components/smartSearchBar/index.tsx

```diff
@@ -1514,13 +1514,23 @@ class SmartSearchBar extends Component<Props, State> {
 
     const cursor = this.cursorPosition;
 
+    const visibleShortcuts = shortcuts.filter(
+      action =>
+        action.hotkeys &&
+        (!action.canRunShortcut ||
+          action.canRunShortcut(this.cursorToken, this.filterTokens.length))
```

Comment:
Should it just be `action.canRunShortcut?.(...)` ?

Reply:
canRunShortcut is optional for when we want to have an shortcut that can run without a condition I guess, originally the arrow key ones had no condition but added some because we also use this method to check whether we render it in the footer or not now

Reply:
should I refactor it for the method to just be required

Reply:
if you think it makes sense sure